src/main/resources/schemas/nanopore-instrument-output.schema.json

```diff
@@ -97,6 +97,20 @@
         }
       ]
     },
+    "fastqgz_file": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/file"
+        },
+        {
+          "properties": {
+            "file_type": {
+              "pattern": ".*fastq.gz$"
```

Comment:
Didn't we want to only have "fastq.gz"?

Reply:
Hm, this should have been reverted. Nice spot!